pkg/database/vercode_test.go

```diff
@@ -288,26 +287,6 @@ func TestVerCodeValidate(t *testing.T) {
 			},
 			Err: ErrInvalidTestType,
 		},
-		{
```

Comment:
I'm a little hesitant about this validation moving into the controller.

Reply:
At the least it isn't duplicated. The date validation relies on config values for maxAge. It's a little odd to have those kinds of dependencies in the model.

We could go all-in and try to get all of the validation, deps and all, in there but this gets us to a good spot for now?